web/src/app/pages/create-survey/create-survey.component.ts

```diff
@@ -35,6 +35,79 @@ import {TaskService} from 'app/services/task/task.service';
 
 import {SurveyLoiComponent} from './survey-loi/survey-loi.component';
 
+export enum SetupPhase {
```

Comment:
Nit: The component is called "Create" survey, but this enum is called "Setup", which might be taken to mean these are referring to two different things. Can we use one or the other consistently? (eg. `CreateSurveyPhase`)?

Reply:
done